src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java

```diff
@@ -39,30 +43,26 @@
     protected Map<String, Object> runFormSpecificTransformation(List<OcrDataField> ocrDataFields) {
         Map<String, Object> transformedCaseData = new HashMap<>();
 
-        mapFullNameToFirstAndLast(OcrFieldName.APPLICANT_FULL_NAME, "applicantFMName", "applicantLName",
-            ocrDataFields, transformedCaseData);
-        mapFullNameToFirstAndLast(OcrFieldName.RESPONDENT_FULL_NAME, "appRespondentFMname", "appRespondentLName",
-            ocrDataFields, transformedCaseData);
+        mapFullNameToFirstAndLast(APPLICANT_FULL_NAME, "applicantFMName", "applicantLName", ocrDataFields, transformedCaseData);
+        mapFullNameToFirstAndLast(RESPONDENT_FULL_NAME, "appRespondentFMname", "appRespondentLName", ocrDataFields, transformedCaseData);
 
-        commaSeparatedEntryTransformer(OcrFieldName.NATURE_OF_APPLICATION, "natureOfApplication2", natureOfApplicationChecklistToCcdFieldNames,
+        commaSeparatedEntryTransformer(NATURE_OF_APPLICATION, "natureOfApplication2", natureOfApplicationChecklistToCcdFieldNames,
             ocrDataFields, transformedCaseData);
-        commaSeparatedEntryTransformer(OcrFieldName.DISCHARGE_PERIODICAL_PAYMENT_SUBSTITUTE, "dischargePeriodicalPaymentSubstituteFor",
+        commaSeparatedEntryTransformer(DISCHARGE_PERIODICAL_PAYMENT_SUBSTITUTE, "dischargePeriodicalPaymentSubstituteFor",
             dischargePeriodicalPaymentSubstituteChecklistToCcdFieldNames, ocrDataFields, transformedCaseData);
 
-        AddressMapper.applyMappings("applicantSolicitor", "solicitorAddress", ocrDataFields, transformedCaseData);
+        applyMappings("applicantSolicitor", "solicitorAddress", ocrDataFields, transformedCaseData);
```

Comment:
I thought aboute removing `AddressMapper` but it actually gives context - `applyMappings` doesn't tell much.